File: reverse-words-string.php
<?php

class Solution {

    /**
     * @param String $s
     * @return String
     */
    function reverseWords($s) {
        $s = explode(' ', $s);
        $ret = [];
        foreach( $s as $a ){
            $a = str_split( $a);
            $a = array_reverse($a);
            $ret[] = implode( '', $a );
        }
        return implode( ' ', $ret );
    }
}

$s = "Let's take LeetCode contest";
$sol = new Solution();
echo $sol->reverseWords($s);

File: reverse-words-string.py
class Solution:
    def reverseWords(self, s: str) -> str:
        print(s)
        temp = s.split(" ")
        result = []
        for word in temp:
            word = word[::-1]
            result.append(word)
        return ' '.join(result)

s = "Let's take LeetCode contest"
sol = Solution()
sol.reverseWords(s)